
subscribers:
  
  queue_size: 5

  image_rgb:
    topic: /kinect2/qhd/image_color_rect

  image_depth:
    topic: /kinect2/qhd/image_depth_rect

  camera_info:
    topic: /kinect2/qhd/camera_info

publishers:
  
  queue_size: 5

  image_rgb:
    topic: /vision_system/vsb/image_rgb_raw

  image_depth:
    topic: /vision_system/vsb/image_depth_raw

  camera_info:
    topic: /vision_system/vsb/camera_info

parameters:
  use_exact_time: true
  image_width: 640
  image_height: 480
  robot_tf: base_link
  kinect_tf: kinect2_link
  offset_x: 0.1315
  offset_y: 0.0020
  offset_z: 1.2120
